be/src/column/column_helper.cpp

```diff
@@ -211,32 +210,29 @@ ColumnPtr ColumnHelper::create_column(const TypeDescriptor& type_desc, bool null
     if (VLOG_ROW_IS_ON) {
         VLOG_ROW << "PrimitiveType " << type << " nullable " << nullable << " is_const " << is_const;
     }
-    if (nullable && is_const) {
```

Comment:
why remove this?

Reply:
This case is handled by the following code.